src/xcode/ENA/ENA/Source/Models/Home/HomeActivateCellConfigurator.swift

```diff
@@ -60,12 +60,14 @@ final class HomeActivateCellConfigurator: CollectionViewCellConfigurator {
 		setupAccessibility(for: cell)
 	}
 
-	func set(newState: RiskDetectionState) {
-		state = newState
-	}
-
 	func setupAccessibility(for cell: ActivateCollectionViewCell) {
 		cell.isAccessibilityElement = true
 		cell.accessibilityIdentifier = Accessibility.StaticText.homeActivateTitle
 	}
 }
+
+extension HomeActivateCellConfigurator: ENStateHandlerUpdating {
+	func updateEnState(_ state: ENStateHandler.State) {
+		self.state = state
```

Comment:
When the internet is turned off this card isnt being updated without scrolling in Homescreen

Reply:
Same for turning on internet again

Reply:
fixed.